src/NServiceBus.AcceptanceTests/EndpointTemplates/EndpointCustomizationConfigurationExtensions.cs

```diff
@@ -9,6 +9,7 @@
 
     public static class EndpointCustomizationConfigurationExtensions
     {
+        [Obsolete("Use the `NServiceBus.AcceptanceTesting.Support.AssemblyScanningExtensions.FilterByTest` extension method instead", false)]
```

Comment:
So if we accept this PR we have to release a new alpha version of master and bring it back to all downstreams. I assume we consider master already "moved forward to next minor version" given that we have branched off into the release branch for v8 right?

Reply:
> I assume we consider master already "moved forward to next minor version" given that we have branched off into the release branch for v8 right?

yes, I'm assuming that this would be released as part of 8.1

> So if we accept this PR we have to release a new alpha version of master and bring it back to all downstream.

I don't see an urgent need to update. It will help to reduce duplication on downstream mid-term and when creating new test projects. I added the obsolete to add some guidance given there have been complaints that att changes aren't documented for TF working on updates. but tbh not sure whether the obsolete would cause more problems due to `WarningsAsErrors` so maybe it should be replaced with a simple comment?

Reply:
Personally, I think the obsolete warnings/errors are the better route...

Reply:
> Personally, I think the obsolete warnings/errors are the better route...

agreed but not sure whether it causes some issues for the TF that will be releasing 8.1 given this change can't be pushed at the moment.